home/models.py

```diff
@@ -1766,6 +1754,12 @@ def clean(self) -> None:
                 ValidationError("All WhatsApp templates need a name.")
             )
 
+        message = self.message
+        try:
+            validate_template_variables(message)
```

Comment:
`validate_template_variables()` already returns a list of variables, so let's use that here instead of collecting them again with a regex below.

Reply:
Done